src/libraries/System.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/AnyRef.cs

```diff
@@ -22,6 +22,12 @@ internal AnyRef(IntPtr jsHandle, bool ownsHandle) : base(ownsHandle)
         {
             SetHandle(jsHandle);
             AnyRefHandle = GCHandle.Alloc(this, ownsHandle ? GCHandleType.Weak : GCHandleType.Normal);
+            if (!ownsHandle)
+            {
+                bool _addRefSucceeded = false;
+                DangerousAddRef(ref _addRefSucceeded);
```

Comment:
Where is the DangerousRelease counterpart?

Reply:
This will be from the JS side.  The owns handle false is used only for the Global object right now.

https://github.com/dotnet/runtime/blob/master/src/mono/wasm/runtime/binding_support.js#L105-L108